### PR TITLE
fix: added onExceeding and onExceeded to be available

### PR DESCRIPTION
**Pull request by dancastillo**
onExceeding and onExceeded to be available
issue #232 
#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by mcollina — COMMENTED** [review]
Could you document those in the README?

**Comment by kibertoad** [comment]
probably TS types need to be updated too?

File: index.d.ts [hunk 1]
```diff
@@ -104,6 +104,8 @@ export interface RateLimitOptions {
     context: errorResponseBuilderContext
   ) => object;
   enableDraftSpec?: boolean;
+  onExceeding?: (req: FastifyRequest) => void;
+  onExceeded?: (req: FastifyRequest) => void;
```
Comment (mcollina) [inline index.d.ts:108]: Could you add a test for the new types? we use `tsd`

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**